code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2152,7 +2152,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	generate_data_info(data)
 
 /datum/reagent/consumable/ethanol/fruit_wine/proc/generate_data_info(list/data)
-	var/minimum_percent = 0.15 //Percentages measured between 0 and 1.
+	var/const/minimum_percent = 0.15 //Percentages measured between 0 and 1.
```

Comment:
you should put a comment explaining why you use const here so in the future people come by and don't accidentally unleash the beast